cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-ui.adoc

```diff
@@ -0,0 +1,316 @@
+. Run the following command to see if an OpenShift Cluster Manager role exists:
++
+[source,terminal]
+----
+$ rosa list ocm-role
+----
+
+. The UI displays the commands to create an OpenShift Cluster Manager role with two different levels of permissions:
++
+* *Basic OpenShift Cluster Manager role:* Allows the OpenShift Cluster Manager to have read only access to the account to check if the roles and policies that are required by ROSA are present before creating a cluster. You will need to manually create the required roles, policies, and OIDC provider using the CLI.
+* *Admin OpenShift Cluster Manager role:* Grants the OpenShift Cluster Manager additional permissions to create the required roles, policies, and OIDC provider for ROSA. Using this makes the deployment of a ROSA cluster quicker since the OpenShift Cluster Manager will be able to create the required resources for you.
++
+To read more about these roles, see the xref:../../../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-ocm-roles-and-permissions_rosa-sts-about-iam-resources[OpenShift Cluster Manager roles and permissions] section of the documentation.
++
+For the purposes of this tutorial, use the *Admin OpenShift Cluster Manager role* for the simplest and quickest approach.
+
+. Copy the command to create the Admin OpenShift Cluster Manager role from the sidebar or switch to your terminal and enter the following command:
++
+[source,terminal]
+----
+$ rosa create ocm-role --mode auto --admin --yes
+----
++
+This command creates the OpenShift Cluster Manager role and associates it with your Red Hat account.
++
+.Example output
++
+[source,terminal]
+----
+I: Creating ocm role
+I: Creating role using 'arn:aws:iam::000000000000:user/rosa-user'
+I: Created role 'ManagedOpenShift-OCM-Role-12561000' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-OCM-Role-12561000'
+I: Linking OCM role
+I: Successfully linked role-arn 'arn:aws:iam::000000000000:role/ManagedOpenShift-OCM-Role-12561000' with organization account '1MpZfntsZeUdjWHg7XRgP000000'
+----
+
+. Click *Step 2: User role*.
+
+=== Other OpenShift Cluster Manager role creation options
+* *Manual mode:* If you prefer to run the AWS CLI commands yourself, you can define the mode as `manual` rather than `auto`. The CLI will output the AWS commands and the relevant JSON files are created in the current directory.
++
+Use the following command to create the OpenShift Cluster Manager role in manual mode:
++
+[source,terminal]
+----
+$ rosa create ocm-role --mode manual --admin --yes
+----
+* *Basic OpenShift Cluster Manager role:* If you prefer that the OpenShift Cluster Manager has read only access to the account, create a basic OpenShift Cluster Manager role. You will then need to manually create the required roles, policies, and OIDC provider using the CLI.
++
+Use the following command to create a Basic OpenShift Cluster Manager role:
++
+[source,terminal]
+----
+$ rosa create ocm-role --mode auto --yes
+----
+
```

Comment:
@jneczypor  Hi, After "Click Step 2 : user role" step, It seems like we are loosing the flow on user-role linking steps. i.e. I see that it explains about the ocm-role. Probably this part needs to be before step 2. Similarly there is no step mentioned "Click  step 3: Account roles". Could you please re-order the sections to make it more clear?